src/controllers/dynakube/oneagent/daemonset/volumes.go

```diff
@@ -75,6 +81,13 @@ func getCSIStorageMount() corev1.VolumeMount {
 	}
 }
 
+func getHttpProxyMount() corev1.VolumeMount {
```

Comment:
```suggestion
func prepareHttpProxyMount() corev1.VolumeMount {

or

func buildHttpProxyMount() corev1.VolumeMount {

```

Reply:
I followed the naming convention from the functions above. We can refactor all of them in a different PR if using `prepare` or `build` makes more sense than `get`.

Reply:
As discussed: I will use `prepare` for the new function and leave the old package with the `get` naming. We will refactor afterwards